org.eclipse.scanning.points/src/org/eclipse/scanning/points/JythonIterator.java

```diff
@@ -17,7 +17,7 @@
 
 	private static final Logger logger = LoggerFactory.getLogger(JythonIterator.class);
 
-	private int               index;
+	private int index;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "index" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)